data/core/macros/utils.cfg

```diff
@@ -685,26 +630,35 @@
                 [/have_unit]
             [/filter_condition]
 
-            [foreach]
-                array=second_unit.attack
-                [do]
-                    {CLEAR_VARIABLE this_item.specials.chance_to_hit}
-
-                    [set_variables]
-                        name=this_item.specials.chance_to_hit
-                        to_variable=this_item.specials.original_chance_to_hit
-                    [/set_variables]
-
-                    {CLEAR_VARIABLE this_item.specials.original_chance_to_hit}
-                [/do]
-            [/foreach]
-
-            [unstore_unit]
-                variable=second_unit
-                find_vacant=no
-            [/unstore_unit]
+             [remove_object]
```

Comment:
Thank you for fixing the indentation above. The indentation from this point to the end is wrong.